packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/RealmObjectHelper.kt

```diff
@@ -295,6 +304,16 @@ internal object RealmObjectHelper {
         }
     }
 
+    internal inline fun <reified T : Any> setSet(
+        obj: RealmObjectReference<out BaseRealmObject>,
+        col: String,
+        set: RealmSet<Any?>,
+        updatePolicy: UpdatePolicy = UpdatePolicy.ALL,
+        cache: ObjectCache = mutableMapOf()
+    ) {
+        // TODO
```

Comment:
Maybe better with an actual `TODO("Set is not supported")`, but I guess it is not actually tested and will be included in the next PR